components/PageHeader.vue

```diff
@@ -3,47 +3,87 @@
     ref="nav-container"
     v-visible-on-scroll
     fluid
-    class="border-bottom mb-3 d-flex py-3 py-lg-1 px-lg-3 flex-column flex-lg-row"
+    class="border-bottom d-flex py-0 px-lg-3 flex-column flex-lg-row align-items-center"
     data-qa="header"
   >
     <header
-      class="col-lg p-0 m-0 text-center text-lg-left navbar-brand d-flex align-items-center justify-content-center justify-content-lg-start flex-column flex-lg-row"
+      class="col p-0 m-0 text-center text-lg-left navbar-brand d-flex align-items-center justify-content-between justify-content-lg-start flex-row"
       role="banner"
       aria-label="Europeana home"
     >
+      <b-button
+        v-show="!showSearch"
+        variant="light"
+        class="navbar-toggle collapsed ml-3 p-0 flex-column align-items-center justify-content-center"
+        aria-label="Show Sidebar"
+        @click="showSidebar = !showSidebar"
+      >
+        <span />
+        <span />
+        <span />
+      </b-button>
       <SmartLink
+        v-show="!showSearch"
```

Comment:
Moving the link and button in to a parent element such as a container or div breaks the styling and complicates the markup, it is quite easy to evenly space the three element in the header on mobile but with the first two wrapped in another element needing to the take up the same space and position the logo centrally and the menu button to the left becomes harder. 

<img width="593" alt="Screenshot 2020-05-26 at 17 06 20" src="https://user-images.githubusercontent.com/4670058/82876152-4b345e00-9f73-11ea-9508-099c3d0f4f39.png">

I can do it, it will require some more css but I think there's a good chance there will be more bugs because of this, is having two v-shows a big issue? I think it might be the better option in this instance

Reply:
as you suggested using a template to wrap them does get past the styling issues I mentioned above but switching between desktop and mobile the search input is hidden now

Reply:
I see your point, that makes sense. Thanks for the clarification.